codemods/transforms/use-resource-imports/README.md

```diff
@@ -0,0 +1,13 @@
+# use-resource-imports codemod
+
+This codemod transforms relevant `react-resource-router` imports to `react-resource-router/resources`
+
+## Usage
+
+```sh
+npm run codeshift -- --transform codemods/transforms/convert-to-resource-imports/index.ts source_file.tsx
```

Comment:
In case some of the previous comments got lost

- https://github.com/atlassian-labs/react-resource-router/pull/200#discussion_r1182163514
- https://github.com/atlassian-labs/react-resource-router/pull/200#discussion_r1182026185